### PR TITLE
feat(ingest): add option to copy URN, fix graphql docs

### PR DESCRIPTION
**Pull request by anshbansal**
This will help in debugging issues a bit faster.

Before
image

After (Copy button Added for both source and execution)
image

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx [hunk 1]
```diff
@@ -343,6 +343,15 @@ export const IngestionSourceList = () => {
             key: 'x',
             render: (_, record: any) => (
                 <ActionButtonContainer>
+                    <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
```
Comment (shirshanka) [inline datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx:346]: This is copying the urn of the ingestion source, or the execution request?

Comment (jjoyce0510) [inline datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx:346]: Let's say Copy Ingestion Source URN specifically

File: datahub-web-react/src/app/ingest/source/IngestionSourceExecutionList.tsx [hunk 2]
```diff
@@ -160,6 +161,17 @@ export const IngestionSourceExecutionList = ({ urn, lastRefresh, onRefresh }: Pr
             key: 'x',
             render: (_, record: any) => (
                 <div style={{ display: 'flex', justifyContent: 'right' }}>
+                    {record.urn && (
+                        <Tooltip title="Copy URN. An URN uniquely identifies an entity on DataHub.">
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/ingest/source/IngestionSourceExecutionList.tsx:165]: Let's say Copy Execution Request Urn specifically. No need for the information after that if this is for debugging.

File: datahub-graphql-core/src/main/resources/ingestion.graphql [hunk 3]
```diff
@@ -19,13 +19,13 @@ extend type Query {
...
   Fetch a specific ingestion source
-  urn: The primary key associated with the ingestion execution request.
+  urn: The primary key associated with the ingestion source.
   """
   ingestionSource(urn: String!): IngestionSource
 
   """
-  Create a request to execute an ingestion job
-  urn: The primary key associated with the ingestion execution request.
+  Get an execution request
```
Comment (jjoyce0510) [inline datahub-graphql-core/src/main/resources/ingestion.graphql:27]: Thanks!

**Review by jjoyce0510 — APPROVED** [review]

**Review by shirshanka — APPROVED** [review]

**Pull request merged**